### DISCUSSION TITLE
Welcome to hublot-app Discussions!

### DISCUSSION THREAD

**Discussion by marcjazz** [Hublot-Tech/hublot-app#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by latioms** [comment]
Hello

**Comment by francsteves** [comment]
Hello !! je viens de vois la structure defini pour le back-end mais j'ai quelque sous vue que nestjs est respecte fortement l'architecture angular ne serait il pas bon de travailler en module de ce fait on ne mélangera pas les choses

**Reply by marcjazz** [reply]
Bien sur que nous travaillerons en module
